jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/cpu/OverallCPULoadHandler.java

```diff
@@ -29,36 +28,32 @@
   private static final String METRIC_NAME = "runtime.jvm.cpu.utilization";
   private static final String DESCRIPTION = "CPU Utilization";
 
-  private final Meter otelMeter;
-  private BoundDoubleHistogram userHistogram;
-  private BoundDoubleHistogram systemHistogram;
-  private BoundDoubleHistogram machineHistogram;
+  private final BoundDoubleHistogram userHistogram;
+  private final BoundDoubleHistogram systemHistogram;
+  private final BoundDoubleHistogram machineHistogram;
 
   public OverallCPULoadHandler(Meter otelMeter) {
```

Comment:
If we revert the removal of init I think we can merge this and follow up on that. Quarkus preinitializing classes was the reason to have an init method - ideally we can find some way to improve, possibly on the Quarkus side, to avoid forcing the pattern on us but I think we can try that later

Reply:
I'm fine with doing that I just want to understand what problem it solves. It would make sense to me if the init methods accepted a `Meter`, so that the metric instruments could be initialized after the metric sdk was setup, but that wasn't happening.